scripts/deploy

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+# Pass in app as either staging or production first -- must match what you seen in git remote -v
+# Examples:
+# ./deploy staging
+# ./deploy staging --force
+# ./deploy production
+
+app=$1
+
+shift
+
+sha=`git rev-parse HEAD 2>/dev/null`
+current_branch=`git symbolic-ref HEAD 2> /dev/null | sed -e 's/refs\/heads\///'`
+
+# echo "current_branch is $current_branch"
+# echo "sha is $sha"
+# echo "app is $app"
+
+if [ "$app" == "" ]; then
+  echo "You must specify a param value for the app, such as production or staging"
+  exit 1
+fi
+
+if [[ "$app" == "production" && "$current_branch" != "master" ]]; then
+  echo "You can only deploy master to production!"
+  exit 1
+fi
+
+echo Running:
+echo git push $app $current_branch:master $@
+git push $app $current_branch:master $@
+
+echo Setting heroku config DEPLOYMENT_SHA
+echo heroku config:set DEPLOYMENT_SHA=$sha --remote $app
+heroku config:set DEPLOYMENT_SHA=$sha --remote $app
+
+# Run migrations always just in case
+heroku run rake db:migrate --remote $app
+heroku restart --remote $app
```

Comment:
@dylangrafmyre Is there a way to somehow set the `DEPLOYMENT_SHA` environment variable on Heroku to the latest commit on deployment automatically? That's all we're really trying to do.